grpc/tests/src/test/contract_api/account/authorized_keys.rs

```diff
@@ -375,18 +424,24 @@ fn should_not_authorize_deploy_with_duplicated_keys() {
     let exec_request_3 = {
         let deploy = DeployItemBuilder::new()
             .with_address(*DEFAULT_ACCOUNT_ADDR)
-            .with_empty_payment_bytes(
-                runtime_args! { ARG_AMOUNT => *DEFAULT_PAYMENT, },
+            .with_empty_payment_bytes(runtime_args! {
+                ARG_AMOUNT => *DEFAULT_PAYMENT,
+            })
+            .with_session_code(
+                CONTRACT_AUTHORIZED_KEYS,
+                runtime_args! {
+                    ARG_KEY_MANAGEMENT_THRESHOLD => Weight::new(0),
+                    ARG_DEPLOY_THRESHOLD => Weight::new(0)
+                },
             )
-            .with_session_code("authorized_keys.wasm", runtime_args! { "key_management_threshold" => Weight::new(0), "deploy_threshold" => Weight::new(0) })
             .with_deploy_hash([3u8; 32])
             .with_authorization_keys(&[
-                key_1, key_1, key_1, key_1, key_1, key_1, key_1, key_1, key_1, key_1,
+                KEY_1, KEY_1, KEY_1, KEY_1, KEY_1, KEY_1, KEY_1, KEY_1, KEY_1, KEY_1,
             ])
             .build();
         ExecuteRequestBuilder::from_deploy_item(deploy).build()
     };
-    // success: identity key weight + key_1 weight + key_2 weight >= deployment
+    // success: identity key weight + KEY_1 weight + KEY_2 weight >= deployment
```

Comment:
Is this comment out of date? I don't see any KEY_2 in this test.

Reply:
Removed